bin/start

```diff
@@ -1,25 +1,19 @@
-#!/usr/bin/env bash
+#!/bin/sh
 
 # used for local development
 
-
 set -e
 
 cd "$(dirname "$0")"/.. || exit 1
 
-function main() {
-  local machine
-  local arch
-  local target
-  machine="$(uname)"
-  arch="$(uname -m)"
-  target="${machine,,}_${arch,,}"
-
-  local
+main() {
+	machine="$(uname)"
+	arch="$(uname -m)"
+	target="$(echo $machine | tr 'A-Z' 'a-z')_$(echo $arch | tr 'A-Z' 'a-z')"
```

Comment:
can you cut this down so the only change is the tr vs the esoteric bash substitution?

i don't want the whitespace changes, changing from bash to sh, or removing the local var declarations